react/features/always-on-top/index.tsx

```diff
@@ -1,14 +1,11 @@
-// @flow
-
 import React from 'react';
 import ReactDOM from 'react-dom';
 
 import AlwaysOnTop from './AlwaysOnTop';
 
 // Render the main/root Component.
-// $FlowExpectedError
 ReactDOM.render(<AlwaysOnTop />, document.getElementById('react'));
 
 window.addEventListener(
     'beforeunload',
-    () => ReactDOM.unmountComponentAtNode(document.getElementById('react')));
+    () => ReactDOM.unmountComponentAtNode(document.getElementById('react') ?? document.body));
```

Comment:
Would it make sense to extract this to a variable and use it in both, avoiding the ??

Reply:
We can't avoid the `??`. By definition `getElementById` returns either an element or `null`. And the problem is that this function requires an element and not `null` (even though we know that the element exists and it will never be null, TS doesn't)